generators/CoreCLRStatefulActor/index.js

```diff
@@ -266,33 +288,44 @@ var ClassGenerator = generators.Base.extend({
       }
     );
     this.fs.copyTpl(
-      this.templatePath('testclient/testscripts/testclient.sh'),
-      this.destinationPath(path.join(appPackage , serviceProjName + 'TestClient' , 'testclient.sh')),
+      this.templatePath('testclient/testscripts/testclient'+extension2),
+      this.destinationPath(path.join(appPackage , serviceProjName + 'TestClient' , 'testclient'+extension2)),
       {
         testClientProjName: testClientProjName
       }
     );
-    this.fs.copyTpl(
-      this.templatePath('main/common/dotnet-include.sh'),
-      this.destinationPath(path.join(appPackage , serviceProjName + 'TestClient' , 'dotnet-include.sh')),
-      {
-      }
-    );
-     if ( this.isAddNewService == false ) {
+    if(is_Linux){
+      this.fs.copyTpl(
+        this.templatePath('main/common/dotnet-include.sh'),
+        this.destinationPath(path.join(appPackage , serviceProjName + 'TestClient' , 'dotnet-include.sh')),
+        {
+        }
+      );
+    }
+    if ( this.isAddNewService == false ) {
       this.fs.copyTpl(
-        this.templatePath('main/deploy/deploy.sh'),
-        this.destinationPath(path.join(appPackage, 'install.sh')),
+        this.templatePath('main/deploy/deploy'+extension1),
+        this.destinationPath(path.join(appPackage, 'install'+extension1)),
         {
           appPackage: appPackage,
           appName: appName,
           appTypeName: appTypeName
         }
       );
+
+      if(is_Windows){
+        this.fs.copyTpl(
+          this.templatePath('main/deploy/predeploy'+ extension1),
```

Comment:
What is this?

Reply:
This was supposed to be used for loading powershell scripts before using service fabric commandlets. But apparently we no more need this and hence this is removed.